docs/troubleshooting.md

```diff
@@ -56,4 +56,31 @@ In short, this driver tries to leave things as it found them.
 You can use the `noReset` capability to adjust this behavior.
 Setting `noReset` to `true` will leave the simulator running at the end of a test session.
 
+## Files generated by test runs
```

Comment:
Probably we can move some of below topic under "device preparation"

Reply:
makes sense. I've created the one, and added a few such as Web Inspector as well. Maybe... we still missed something (I haven't confirmed well yet)